aep/general/example.oas.yaml

```diff
@@ -354,6 +354,7 @@
                     'in': 'path',
                     'required': true,
                     'schema': { 'type': 'string' },
+                    'x-aep-resource-reference': { 'resource': 'Publisher' },
```

Comment:
these files are autogenerated now. I think you should modify aep-lib-go to add the new field: https://github.com/aep-dev/aep-lib-go/blob/main/pkg/api/openapi.go#L326. Or file an issue.

Reply:
https://github.com/aep-dev/aep-lib-go/pull/1

Reply:
How is this file autogenerated?

Reply:
here's a PR documenting it! https://github.com/aep-dev/aeps/pull/258. Since I'm sure it'll come up a lot.